pkgs/development/python-modules/torchvision/bin.nix

```diff
@@ -25,7 +26,7 @@ in buildPythonPackage {
 
   src = fetchurl srcs."${stdenv.system}-${pyVerNoDot}" or unsupported;
 
-  disabled = !(isPy37 || isPy38 || isPy39);
+  disabled = !(isPy37 || isPy38 || isPy39 || isPy310);
```

Comment:
```suggestion
  disabled = pythonOlder "3.7 || pythonAtLeast "3.10";
```

Reply:
I think this would need to be 
```
disabled = pythonOlder "3.7 || pythonAtLeast "3.11"
```
or
```
disabled = !(pythonAtLeast "3.7" && pythonOlder "3.10")
```

That is, python 3.10 is supported now.

(Seems vaguely debatable to me to reference non-existing python versions)

Reply:
I also find the list of supported versions to be a bit more readable personally.